src/templates/homepage/InfopicLeftSection.jsx

```diff
@@ -12,25 +12,16 @@ import { fetchImageURL } from "utils"
  */
 
 const TemplateInfopicLeftSection = (
-  {
-    title,
-    subtitle,
-    description,
-    button,
-    sectionIndex,
-    imageUrl,
-    imageAlt,
-    siteName,
-  },
+  { title, subtitle, description, button, sectionIndex, image, alt, siteName },
```

Comment:
nit: 
- Why did we rename `imageUrl` to `image`? Is it because `imageUrl` is actually the image file name? 
- if `alt` is the alt image url that can be loaded, is it better to name it as `altUrl`?